app/src/main/java/org/jellyfin/androidtv/util/apiclient/ApiClientExtensions.kt

```diff
@@ -45,10 +48,26 @@ suspend fun ApiClient.markUnplayed(itemId: String, userId: String) : UserItemDat
 	})
 }
 
-
 suspend fun ApiClient.updateFavoriteStatus(itemId: String, userId: String, isFavorite: Boolean) : UserItemDataDto? = suspendCoroutine { continuation ->
 	UpdateFavoriteStatusAsync(itemId, userId, isFavorite, object : Response<UserItemDataDto>() {
 		override fun onResponse(response: UserItemDataDto?) { continuation.resume(response!!) }
 		override fun onError(exception: Exception?) { continuation.resume(null) }
 	})
 }
+
+suspend fun ApiClient.getSimilarItems(item: BaseItem): List<BaseItem>? = suspendCoroutine {continuation ->
+	val query = SimilarItemsQuery()
+	query.setUserId(TvApp.getApplication().currentUser.id)
+	query.setId(item.id)
+	query.setLimit(10)
```

Comment:
I'd add the limit as parameter with 10 as default value

Maybe increase it to 25 or something too

Reply:
Sure, can do that